src/golang/lib/collections/operator/connector/extract_params.go

```diff
@@ -11,6 +11,7 @@ type ExtractParams interface {
 type RelationalDBExtractParams struct {
 	GithubMetadata *gh_types.GithubMetadata `json:"github_metadata"`
 	Query          string                   `json:"query"`
+	Queries        []string                 `json:"queries"`
```

Comment:
I assume we made them two fields instead of one list field so that we wouldn't have to do a migration. How hard would a migration be? This seems like unnecessary tech debt to me, I'd much rather reason about a single list.